include/arkode/arkode_erkstep.h

```diff
@@ -67,6 +69,12 @@ SUNDIALS_EXPORT int ERKStepGetTimestepperStats(
   void* arkode_mem, long int* expsteps, long int* accsteps,
   long int* step_attempts, long int* nfevals, long int* netfails);
 
+/* Adjoint solver functions */
+SUNDIALS_EXPORT
+int ERKStepCreateAdjointStepper(void* arkode_mem, SUNAdjRhsFn adj_f,
+                                sunrealtype tf, N_Vector sf,
+                                SUNAdjointStepper* adj_stepper_ptr);
```

Comment:
Should we pass in the context explicitly to the constructor?

Reply:
I dont think we should because it should probably use the same one as the original arkode_mem.